.buildkite/scripts/lib/packer.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Localizing the core "logic" (such as it is) for building our AMI. We
+# use this both to build locally and in CI.
+build_ami() {
+    # Example usage:
+    # PACKER_VARS="-var build_ami=false" build_ami
+
+    # shellcheck disable=SC2086
+    packer build ${PACKER_VARS:-} packer/nomad-server
+    # shellcheck disable=SC2086
+    packer build ${PACKER_VARS:-} packer/grapl-service
```

Comment:
I think it would be worth investing some time to parameterize this `build_ami` function so it takes the path to a template to build. This, in turn, would allow us to build these in parallel (either locally in two different terminals, or in CI with two separate jobs).

A single AMI takes long enough to build as it is; doubling that (and beyond, once we have more AMIs) is going to make these pipeline runs unbearably long.

Reply:
At that point I'm not sure what value `PACKER_VARS="-var asdf=123" build_ami packer/grapl_service` adds over just `packer build -var asdf=123 packer/grapl_service`

Reply:
re: doing these two AMIs in parallel:
1. i agree
2. would you accept "let me file a ticket and it's on my plate immediately after this PR committed"
"building AMIs faster" doesn't get us closer to nomad

Reply:
ticket: https://app.zenhub.com/workspaces/grapl-6036cbd36bacff000ef314f2/issues/grapl-security/issue-tracker/616